NeuralNet/evaluateNN.py

```diff
@@ -133,3 +133,29 @@ def parse_input(filename):
 results = model.evaluate(test_data, test_labels)
 
 print(results)
+
+def makePrediction(sentence):
+    (indices, wordlist, poslist) = db_pull.in_pipe([sentence])
```

Comment:
id suggest changing these variable names cus they conflict with same names in a bigger scope

Reply:
I'm not sure where the conflict is. Can you change them?